json-definitions/v3/tech-record/put/complete/request/index.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "Tech Record PUT Request Complete Car Schema",
+  "type": "object",
+  "additionalProperties": true,
+  "required": [
+    "techRecord_vehicleSubclass"
+  ],
+  "allOf": [
+    {
+      "$ref": "../response/index.json",
+      "required": [
+        "vin",
+        "primaryVrm",
+        "trailerId",
+        "techRecord_recordCompleteness",
+        "techRecord_euVehicleCategory",
+        "techRecord_vehicleType",
+        "techRecord_manufactureYear",
```

Comment:
So, I hate to do this especially since I was the one who suggested it, however I think the extending the required, might not work.
The issue is just with the `null`. Let's take `manufactureYear` It can be null in a skeleton, but we don't want it to be `null` in a complete (I don't actually know if this is the case, but just an example). I don't know if there is a way of effectively doing a "spread" operator. 

Just for illustrative purposes this is what I mean (obviously this would not work in json):
```js
const skeleton = {some_property: ["string", "null"], another_property: "string"}

const complete = {...skeleton, some_property: "string"}
```

I tried doing something like this (where `skeleton` would be as above):

```json
{
  "name": "complete"
  "allOf": [
   {"$ref": "skeleton"}
  ],
  "properties": {
    "some_property": {
      "type": "string"
    }
  }
}

```
however that did not seem to work.